enterprise/server/backends/openai/BUILD

```diff
@@ -6,7 +6,7 @@ go_library(
     importpath = "github.com/buildbuddy-io/buildbuddy/enterprise/server/backends/openai",
     visibility = ["//visibility:public"],
     deps = [
-        "//cli/log",
```

Comment:
Maybe we should add a visibility rule to disallow cli imports from //server/...

Reply:
Will send a separate PR for that, also removed the public visibility of this rule and the vertex one so it falls back to the default //enterprise subpackages